manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760
+PackageLocale: en-US
+Publisher: Arturia
+PackageName: Arturia Software Center
+License: Proprietary
+Copyright: (c) 2014 Arturia
+ShortDescription: Arturia Software Center Setup

```

```suggestion
ShortDescription: The Arturia Software Center · Is an elegant, simple solution to help you install, activate, and update your Arturia software instruments.
```
ShortDescription should be more descriptive